packages/components/src/color-indicator/types.ts

```diff
@@ -0,0 +1,12 @@
+/**
+ * External dependencies
+ */
+import type { CSSProperties } from 'react';
+
+export type ColorIndicatorProps = {
+	/**
+	 * The color of the indicator. Any value from the background property is
```

Comment:
Maybe add a little tweak for clarity here?

```suggestion
	 * The color of the indicator. Any value from the CSS `background` property is
```

Reply:
Makes sense clearify it 👍